3-enrich/emr-etl-runner/spec/snowplow-emr-etl-runner/cli_spec.rb

```diff
@@ -239,9 +239,22 @@ def resource(name)
       Cli.validate_and_coalesce(a, c)
     end
 
-    it 'should accept stream-mode config' do
+    it 'should accept stream enrich mode config' do
       Cli.validate_and_coalesce(a, s)
     end
-  end
 
+    it 'should reject --skip staging in stream enrich mode' do
+      a[:skip] = [ 'staging' ]
+      expect {
+        Cli.validate_and_coalesce(a, s)
+      }.to raise_exception( ConfigError, "cannot skip staging nor enrich in stream enrich mode. Either skip staging_stream_enrich or resume from shred")
+    end
+
+    it 'should reject --resume-from enrich in stream enrich mode' do
+      a[:resume_from] = 'enrich'
+      expect {
+        Cli.validate_and_coalesce(a, s)
+      }.to raise_exception( ConfigError, "cannot resume from enrich in stream enrich mode")
```

Comment:
you're missing a couple of tests for archive_raw and skip enrich also nit: `to raise_exception(Config...`